tests/utils.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python
```

Comment:
```suggestion
#!/usr/bin/python3
```

Reply:
Thanks! Should not be coding this late :)